entity-framework/core/modeling/relationships.md

```diff
@@ -92,13 +92,13 @@ See [Cascade Delete](../saving/cascade-delete.md) for more details about the dif
 
 ## Data Annotations
 
-There are two data annotations that can be used to configure relationships, `[ForeignKey]` and `[InverseProperty]`.
+There are two data annotations that can be used to configure relationships, `[ForeignKey]` and `[InverseProperty]`. These are available in the `System.ComponentModel.DataAnnotations` assembly.
 
 ### [ForeignKey]
 
 You can use the Data Annotations to configure which property should be used as the foreign key property for a given relationship. This is typically done when the foreign key property is not discovered by convention.
 
-[!code-csharp[Main](../../../samples/core/Modeling/DataAnnotations/Samples/Relationships/ForeignKey.cs?name=Entities&highlight=17)]
+[!code-csharp[Main](../../../samples/core/Modeling/DataAnnotations/Samples/Relationships/ForeignKey.cs?name=Entities&highlight=30)]
```

Comment:
This doesn't highlight the right thing or show any namespace in the preview. I think it is because the reference is using a region.

Reply:
OK. I guess I'll have to find something other than VSCode to test with - as this worked for me. What do you use?

Reply:
I looked at the preview site that gets published. There is a link in the checks section, and also an email when you commit. Anyway, I am pretty sure that the issue is that name=Entity in the address means to extract the Entity region, so you have to either remove that or make sure what you want to show is in the region, and that the highlight value represents the line number inside the region.

Reply:
Yep. That was the problem. Fixed